bedrock/firefox/templates/firefox/releases/notes.html

```diff
@@ -90,13 +90,13 @@
               <div class="mzp-c-navigation-menu">
                 <nav class="mzp-c-menu mzp-is-basic">
                   <ul class="mzp-c-menu-category-list">
-                    {% if release.channel == 'Release' and equivalent_release_url %}
+                    {% if release.channel == 'Release' and release.product == 'Firefox for Android' and equivalent_release_url %}
                       <li class="mzp-c-menu-category"><a class="mzp-c-menu-title" href="{{ equivalent_release_url }}">{{ _('Desktop') }}</a></li>
                     {% else %}
                       <li class="mzp-c-menu-category"><a class="mzp-c-menu-title" href="{{ url('firefox.notes') }}">{{ _('Desktop') }}</a></li>
```

Comment:
We don't really need a link at all if we're already looking at desktop notes right? But I guess this would take you to the latest notes if you were looking at an older one. Hmm... This does seem like an improvement regardless.